### PR TITLE
Fixes #15514:  Make certificate verification in HTTP calls configurable - techniques

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15514

### CODE REVIEW

File: share/lib/common.sh [hunk 1]
```diff
@@ -168,6 +168,7 @@ if [ -f "${RUDDER_JSON}" ]; then
   AGENT_RUN_INTERVAL=$(rudder_json_value 'AGENT_RUN_INTERVAL')
   RUDDER_NODE_CONFIG_ID=$(rudder_json_value 'RUDDER_NODE_CONFIG_ID')
   RUDDER_SYSLOG_PROTOCOL=$(rudder_json_value 'RUDDER_SYSLOG_PROTOCOL')
+  RUDDER_VERIFY_CERTIFICATES=$(rudder_json_value 'RUDDER_VERIFY_CERTIFICATES')
```
Comment (ncharles) [inline share/lib/common.sh]: does it have a defaut value ?

File: share/lib/common.sh [hunk 2]
```diff
@@ -182,3 +183,11 @@ then
 else
   FULL_COMPLIANCE=1
 fi
+
+if [ "${RUDDER_VERIFY_CERTIFICATES}" = "true" ] 
+then
+  CERTIFICATE_OPTION="--insecure"
```
Comment (ncharles) [inline share/lib/common.sh]: i think you want the opposite

Comment (amousset) [reply share/lib/common.sh]: indeed

**Comment by amousset** [comment]
PR updated with a new commit

File: share/lib/common.sh (hunk 1 above)
Comment (amousset) [reply share/lib/common.sh]: No but we compare it to true so it should work

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**